app/services/events/stores/postgres_store.rb

```diff
@@ -38,6 +38,19 @@ def last_event
         events.last
       end
 
+      def grouped_last_event
+        groups = sanitized_grouped_by
+
+        sql = events
+          .reorder(Arel.sql((groups + ['events.timestamp DESC, created_at DESC']).join(', ')))
+          .select(
+            "DISTINCT ON (#{groups.join(', ')}) #{groups.join(', ')}, events.timestamp, (#{sanitized_propery_name})::numeric AS value",
```

Comment:
Layout/LineLength: Line is too long. [135/120] (https://rubystyle.guide#max-line-length)